main/security.bs

```diff
@@ -3,6 +3,14 @@ Security Considerations {#security}
 
 Issue: Write Security Considerations section.
 
+Data pods SHOULD use TLS connections
+to protect the contents of requests and responses
+from eavesdropping and modification by third parties.
+Regular TCP connections MAY be used
```

Comment:
TLS connections are also regular TCP connections, so I think we should instead say something like "Unsecured connections that do not use TLS"